src/naming.adoc

```diff
@@ -74,16 +74,16 @@ if it follows a number. For example, "rv32i2p2" means version 2.2 of
 RV32I, whereas "rv32i2_p2" means version 2.0 of RV32I with version 2.0
 of the P extension.
 
-=== Additional Standard Extension Names
+=== Additional Standard Unprivileged Extension Names
 
-Standard extensions can also be named using a single "Z" followed by
+Standard unprivileged extensions can also be named using a single "Z" followed by
 an alphabetical name and an optional version number. For example,
 "Zifencei" names the instruction-fetch fence extension described in
 <<zifencei>>; "Zifencei2" and
 "Zifencei2p0" name version 2.0 of same.
 
 The first letter following the "Z" conventionally indicates the most
-closely related alphabetical extension category, IMAFDQCVH. For the
+closely related alphabetical extension category, IMAFDQLCBKJTPVH. For the
```

Comment:
Is this basically a revert of https://github.com/riscv/riscv-isa-manual/commit/f5f9c27010b69a015958ffebe1ac5a34f8776dff

Reply:
For this, I think the answer is Yes. I reviewed the comments in that commit and understand what happened. If we can’t decide in time, I think the other part of this patch that clarifies the Z* is unprivileged and the order of Zicsr, Zifencei, and Ztso should also be removed to avoid confusion for readers.

Reply:
Note `H` is _not_ an unprivileged extension so does not belong here.

Reply:
unfortunately the order of H needs to be listed because it governs the sorting of Zhinx Zhinxmin

Reply:
Yeah, those probably should've been named Zfhinx... Seems like the H is not in the ideal place in the list, then?  Shouldn't it be somewhere around Q?

Reply:
> Note `H` is _not_ an unprivileged extension so does not belong here.

Will Zh* exist someday for hypervisor additional extension? If yes, I think "where to place it" should be defined.

Reply:
No, hypervisor additional extensions use the `Sh` prefix (like `Sm` for machine or `Ss` for supervisor)